com.unity.shadergraph/Editor/Data/Graphs/AbstractShaderProperty.cs

```diff
@@ -126,7 +126,7 @@ internal virtual string GetPropertyAsArgumentStringForVFX(string precisionString
 
         public virtual string GetPropertyTypeString()
         {
-            string depString = $" (Deprecated{(ShaderGraphPreferences.allowDeprecatedBehaviors ? " V" + sgVersion : "")})";
+            string depString = $" (Upgradable{(ShaderGraphPreferences.allowDeprecatedBehaviors ? " V" + sgVersion : "")})";
```

Comment:
Why not make this a const static here that can be used everywhere else for consistency?

Reply:
Hmm...  probably better to make a function that will append the version string, then we could use that function everywhere, instead of having the logic in two places (and hardcoded in a third)